src/plugins/console/public/lib/es/es.ts

```diff
@@ -36,6 +36,9 @@ export function send(method: string, path: string, data: any) {
 
   const options: JQuery.AjaxSettings = {
     url: '../api/console/proxy?' + stringify({ path, method }, { sort: false }),
+    headers: {
```

Comment:
It seems like in the legacy platform, angular takes care of that.

Reply:
Without this header, the request doesn't go through at all. When switching to `core.http` here, it should become unnecessary, because core takes care of that as well.

Reply:
Alright, thanks for clarification!